src/state.rs

```diff
@@ -177,14 +177,28 @@ impl State {
             }
             KeyPress {
                 keysym: keysyms::XKB_KEY_Return,
+                ctrl,
                 ..
             } => {
                 let info = EvalInfo {
                     index: self.filtered_lines.index(self.selected_item),
                     subindex: self.selected_subitem,
                     input_value: self.input_buffer.parsed_input(),
                 };
-                self.inner.eval(info);
+                if ctrl {
+                    match unsafe { nix::unistd::fork() } {
+                        Ok(v) => {
+                            if v.is_child() {
+                                self.inner.eval(info);
+                            } else {
+                                return false;
+                            }
+                        }
+                        Err(e) => log::debug!("fork() error: {:?}", e),
```

Comment:
```suggestion
                        Err(e) => log::error!("fork() error: {}", e),
```

Reply:
Fix for that silly copy-pasta mistake is included in [new commit](https://github.com/l4l/yofi/pull/133/commits/a18dfab71d104525a0da0c9224460f2362ca9884). Thanks!